pyroma/tests.py

```diff
@@ -1,17 +1,14 @@
-import unittest
-import os
 import collections
+import json
+import mock
```

Comment:
```suggestion
from unittest import mock
```

Are you intentionally using the bleeding-edge third-party version of `mock` here, or will the stdlib `unittest.mock` suffice and avoid an extra test dependency?

Reply:
Other than the aforementioned error finding `distribute.json`, which could theoretically obscure a later failure, this substitution appears to work just fine in my testing.

Reply:
Oh, right, I forgot that mock is in stdlib now.